### PR TITLE
Users/pmykhalevych/windows add attention mesh dll

### PR DESCRIPTION
**Pull request by pavloMykhalevych**
Change list:
- add a visualization of points in face_mesh_cpu
- change face_landmark_front_side_model_cpu_with_face_counter and face_landmark_side_model_cpu to work with attention
- add the possibility to turn off/on attention model
- now we have 478 points if with_attention is set to true(else 468)

### CODE REVIEW

File: mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp [hunk 1]
```diff
@@ -30,6 +47,8 @@ MPFaceMeshDetector::InitFaceMeshDetector(int numFaces,
   // Prepare graph config.
   auto preparedGraphConfig = absl::StrReplaceAll(
       graphConfig, {{"$numFaces", std::to_string(numFaces)}});
+  preparedGraphConfig = with_attention ? absl::StrReplaceAll( preparedGraphConfig, { {"$with_attention", "true"} }) :
+      absl::StrReplaceAll( preparedGraphConfig, { {"$with_attention", "false"} });
```
Comment (maletsden) [inline mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp:51]:
```suggestion
  preparedGraphConfig = absl::StrReplaceAll(preparedGraphConfig, { {"$with_attention",  with_attention ? "true" :  "false"} });
```

File: mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.h [hunk 2]
```diff
@@ -79,8 +86,12 @@ extern "C" {
...
 
 DLLEXPORT MPFaceMeshDetector *
-MPFaceMeshDetectorConstruct(int numFaces, const char *face_detection_model_path,
-                            const char *face_landmark_model_path);
+MPFaceMeshDetectorConstruct(int numFaces,
+    const char *face_detection_model_path,
+    const char *face_landmark_model_path,
+    bool with_attention = true,
+    const char* face_landmark_model_with_attention_path = "mediapipe/modules/face_landmark/face_landmark_with_attention.tflite"
+    );
```
Comment (maletsden) [inline mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.h:94]: Please, add default values for `face_detection_model_path` and `face_landmark_model_path` as well.

Also, maybe change the order, variables first, then model paths.

Btw:
```suggestion
MPFaceMeshDetectorConstruct(int numFaces,
    const char *face_detection_model_path,
    const char *face_landmark_model_path,
    bool with_attention = true,
    const char* face_landmark_model_with_attention_path = "mediapipe/modules/face_landmark/face_landmark_with_attention.tflite");
```

File: mediapipe/modules/face_landmark/BUILD [hunk 3]
```diff
@@ -47,14 +47,18 @@ mediapipe_simple_subgraph(
...
         "//mediapipe/calculators/core:split_vector_calculator",
         "//mediapipe/calculators/tensor:image_to_tensor_calculator",
         "//mediapipe/calculators/tensor:inference_calculator",
         "//mediapipe/calculators/tensor:tensors_to_floats_calculator",
         "//mediapipe/calculators/tensor:tensors_to_landmarks_calculator",
+		"//mediapipe/calculators/tflite:tflite_custom_op_resolver_calculator",
         "//mediapipe/calculators/util:landmark_projection_calculator",
         "//mediapipe/calculators/util:thresholding_calculator",
+		"//mediapipe/framework/tool:switch_container",
     ],
```
Comment (maletsden) [inline mediapipe/modules/face_landmark/BUILD:62]: The wrong tabulation.
As I checked you didn't add ":face_landmarks_model_loader", please, add it as well.

**Review by maletsden — CHANGES_REQUESTED** [review]
Please, resolve the comments and merge this PR.

File: .gitignore [hunk 4]
```diff
@@ -4,3 +4,4 @@ mediapipe/MediaPipe.tulsiproj/*.tulsiconf-user
 mediapipe/provisioning_profile.mobileprovision
 .configure.bazelrc
 .user.bazelrc
+.vs
```
Comment (maletsden) [inline .gitignore:7]: Please, remove this change. (It's ok to have it locally, but then just don't commit).

File: WORKSPACE [hunk 5]
```diff
@@ -212,7 +212,7 @@ new_local_repository(
 new_local_repository(
     name = "windows_opencv",
     build_file = "@//third_party:opencv_windows.BUILD",
-    path = "C:\\opencv\\build",
+    path = "D:\\opencv\\build",
```
[the remark this answers is not in the document]
Comment (pavloMykhalevych) [reply WORKSPACE]: Change

File: mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp (hunk 1 above)
Comment (pavloMykhalevych) [reply mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp]: Done

File: mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.h (hunk 2 above)
Comment (pavloMykhalevych) [reply mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.h]: Done

**Review by maletsden — APPROVED** [review]

**Pull request merged**